src/components/Details/carsDetails.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useAuthHeader } from 'react-auth-kit';
+import { useParams, useDispatch } from 'react-router-dom';
+// import { useParams, useNavigate } from 'react-router-dom';
+import { AiOutlineRight } from 'react-icons/ai';
+import Loader from '../Loader/Loader';
+import { fetchCars } from '../../redux/cars/cars';
+
+const DetailsCarScreen = () => {
+  const [loading, setLoading] = useState(true);
+  const dispatch = useDispatch();
+  const authHeader = useAuthHeader();
+  const { id } = useParams();
+  const carDetails = useSelector((state) => state.cars);
+  const cars = carDetails;
```

Comment:
Kindly use object destructuring to get the cars value from the carDetails object
```suggestion
  const { cars } = carDetails;
```